src/app/info/components/ChartGovernanceActiveDelegates.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { useEffect, useRef, useState } from "react";
+import {
+  Area,
+  CartesianGrid,
+  ComposedChart,
+  Line,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from "recharts";
+import ChartTabs from "./ChartTabs";
+import useTenantColorScheme from "@/hooks/useTenantColorScheme";
+import type { MetricTimeSeriesValue } from "@/lib/types";
+import { FREQUENCY_FILTERS } from "@/lib/constants";
+
+interface Props {
+  getData: (
+    metric: string,
+    frequency: string
+  ) => Promise<{ result: MetricTimeSeriesValue[] }>;
+}
+
+type ChartData = {
+  active: string;
+  large: string;
+  date: string;
+};
+
+const ChartGovernanceActiveDelegates = ({ getData }: Props) => {
+  const { primary } = useTenantColorScheme();
```

Comment:
I'm doing some work to add custom colors per tenant so we will be able to use that down the line